README.rst

```diff
@@ -1,23 +1,53 @@
-==============
 More Itertools
 ==============
 
 .. image:: https://coveralls.io/repos/github/erikrose/more-itertools/badge.svg?branch=master
   :target: https://coveralls.io/github/erikrose/more-itertools?branch=master
 
-I love itertools; it's one of the most beautiful, composable standard libs.
-Whenever I have an iteration problem, there's almost always an itertools
-routine that fits it perfectly. Sometimes, however, neither itertools nor the
-recipes included in its docs do quite what I need.
+Python's ``itertools`` library is a gem - you can compose elegant solutions
+for a variety of problems with the functions it provides. In ``more-itertools``
+we collect additional building blocks, recipes, and routines for working with
+Python iterables.
+
+Getting started
+===============
+
+To get started, install the library with `pip <https://pip.pypa.io/en/stable/>`_:
+
+.. code-block:: shell
+
+    pip install more-itertools
```

Comment:
Eh. I think you'll need some more help than we can provide if you're attempting to import something with a `-` in the name.